bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,173 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import hero_image, join_waitlist_button, vpn_nav_platform_cta, vpn_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}Mozilla VPN: Protect Your Entire Device{% endblock %}
```

Comment:
String should be in Fluent?

Reply:
Yes. Great catch.

@rraue I tagged you in the google doc also, but do we have specific title/meta descriptions for each page?

Reply:
There should be, if not I get them in.

Reply:
I got direction from @rraue. All good!